pkg/reconciler/apis/apibinding/apibinding_reconcile.go

```diff
@@ -541,5 +538,27 @@ func generateCRD(schema *apisv1alpha1.APIResourceSchema) (*apiextensionsv1.Custo
 		crd.Spec.Versions = append(crd.Spec.Versions, crdVersion)
 	}
 
+	if len(schema.Spec.Versions) > 1 && schema.Spec.Conversion == nil {
```

Comment:
do we have validation in place that check this? Here we are pretty late in the process.

Reply:
We don't at the moment. This should either be in admission or checked through a CEL expression in the APIResourceSchema. What do you think?

Reply:
Not sure if one can put a Kubebuilder validation on the spec field? Never tried.

Reply:
Added this in the latest commit and verified by trying to create a CRD.